pages/common/comm.md

```diff
@@ -0,0 +1,23 @@
+# comm
+
+> Select or reject lines common to two files. Both files must be sorted.
+
+- Produce three-tab separated columns: lines only in first file, lines only in second file and common lines.
```

Comment:
That's just a typo IMO, I believe he meant to put the hyphen between "tab" and "separated" but instead put it between "three" and "tab" (see discussion above).